packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -49,6 +54,33 @@ const PUBLIC_TOPICS = {
   account: ['Staking Account holder status', M.any()],
 };
 
+// UNTIL https://github.com/cosmology-tech/telescope/issues/605
+/**
+ * @param {Any} x
+ * @returns {AnyJson}
+ */
+const toJSON = x => /** @type {AnyJson} */ (Any.toJSON(x));
```

Comment:
consider including "Any" in the fn name
```suggestion
const toAnyJson = x => /** @type {AnyJson} */ (Any.toJSON(x));
```